runtime/kusama/src/weights/pallet_balances_balances.rs

```diff
@@ -0,0 +1,154 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
```

Comment:
It seems that we are not using the correct weight file for Kusama follow up for https://github.com/paritytech/polkadot/issues/7408